src/site/apt/sessions/session-access-grants.apt.vm

```diff
@@ -3,6 +3,124 @@ Access Grants
     {{{https://docs.inrupt.com/ess/latest/security/access-requests-grants/}Access Grants}} provide an alternative way to give access to your Solid resources. The Inrupt Java Client Libraries
     provide tooling for working with Access Grants.
 
+    To use {{{https://docs.inrupt.com/ess/latest/security/access-requests-grants/}Access Grants}}, the resources one wants to access must have {{{https://docs.inrupt.com/ess/latest/security/access-requests-grants/#access-grants-acp}ACP that enables the use of access grants}} and the Authorization Service must be able to exchange an access grant for a token (for example such as a {{{https://docs.inrupt.com/ess/latest/services/service-uma/}UMA Service}}).
+
+* ACP that enables the use of Access Grants
+
+    One needs to edit the ACP of the resource the access grant will be used for. 
+    
+** Manually adding triples to ACP
+    
+    This can be done manually through using a resource management interface such as {{{https://podpro.dev/} Pod Pro}}.
+
+    The next example, adds a new access control that matches for <<<http://www.w3.org/ns/solid/vc#SolidAccessGrant>>>> and allows Read and Write for a Solid Container and its contents. 
```

Comment:
This is just an ACR. It doesn’t “add” anything, it just contains a set of instructions for ACP engines.

It could be PUT to an existing ACR which would just override that ACR (inherited ACs would still apply).